resources/views/layouts/html.blade.php

```diff
@@ -10,20 +10,20 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <!--  Links  -->
-    <link href="{{asset("assets/css/styles.css")}}" rel="stylesheet">
+    <link href="{{asset("assets/css/style.css")}}" rel="stylesheet">
     <link rel="shortcut icon" href="{{asset("assets/img/favicon.png")}}" type="image/png">
 
     <!--  Bootstrap5 CSS  -->
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous"></script>
 
     <script src="https://kit.fontawesome.com/6b5185c021.js" crossorigin="anonymous"></script>
-
+    @stack('scripts')
     <script src="@yield('js')"></script>
```

Comment:
Does we need this line, if @stack already here?

Reply:
We don't. Let's do a git blame? 👺